clarify sdt doc (#2469)
apache/iotdb: docs/UserGuide/Concept/SDT.md

* CompDev (Compression Deviation)

CompDev is the most important parameter in SDT that represents the maximum difference between the 

> current sample and the current linear trend. Default value is 0.

Author: if use SDT and not set compDev or compDev is set to < 0, will throw error..

> * CompMinTime (Compression Minimum Time Interval)

Reviewer: The code should also be modified according to the documentation.

CompMinTime is used for noisy reduction. CompMinTime measures the time distance between two stored data points, 
if the current point's time to the last stored point's time distance is smaller than or equal to CompMinTime, 

> current point will NOT be stored regardless of compression deviation. Default value is 0 and time unit is ms.

Reviewer: What if we don't set CompMinTime and CompMaxTime ?

> if the current point's time to the last stored point's time distance is smaller than or equal to CompMinTime, 
> current point will NOT be stored regardless of compression deviation. Default value is -2147483647 and time unit is ms.

Reviewer 2: ```suggestion
if the time interval between the current point and the last stored point is less than or equal to this value, current point will NOT be stored regardless of compression deviation. 
The Default value is -2147483647 with time unit ms.
```

> CompMinTime is used for noisy reduction. CompMinTime measures the time distance between two stored data points, 
> if the current point's time to the last stored point's time distance is smaller than or equal to CompMinTime, 
> current point will NOT be stored regardless of compression deviation. Default value is -2147483647 and time unit is ms.

Reviewer 2: ```suggestion
CompMinTime is a parameter measures the time distance between two stored data points, which is used for noisy reduction.
If the time interval between the current point and the last stored point is less than or equal to its value, current point will NOT be stored regardless of compression deviation. 
The Default value is -2147483647 with time unit ms.
```
